src/commands/java_restore_maven_artifacts.yml

```diff
@@ -0,0 +1,18 @@
+description: >
+  Restore cached maven artifacts.
+  Works with monorepo as well as single-app-repo.
+
+parameters:
+  maven_cache_key_prefix:
+    description: "Prefix for the cache-key (used in combination with checksum of pom.xml); no-op with blank prefix"
+    type: string
+    default: ""
```

Comment:
I wonder if prefix default value can be app name?

Reply:
see [above](https://github.com/ricardo-ch/ricardo-orbs/pull/17#discussion_r914538969) - IMHO it does not make sense